app/models/institution_builder.rb

```diff
@@ -502,4 +503,30 @@ def self.add_extension_campus_type(version_number)
     SQL
     Institution.connection.update(str)
   end
+
+  def self.build_institution_programs(version_number)
+    str = <<-SQL
+      INSERT INTO institution_programs (
+        facility_code,
+        program_type,
+        description,
+        full_time_undergraduate,
+        graduate,
+        full_time_modifier, length_in_hours,
+        version
+      )
+      SELECT
+        facility_code,
+        program_type,
+        description,
+        full_time_undergraduate,
+        graduate,
+        full_time_modifier, length,
```

Comment:
Nitpick: can we put these on two separate lines like the others?

Reply:
Yes!  Sorry, I was just getting addicted to green code climate checkboxes